README.md

```diff
@@ -73,6 +73,7 @@ List of the remote software companies hiring in Europe
 |StackOverflow 🇺🇸 |Online community|🖥 C#, .NET, TypeScript| ❌ | ✅ |[ℹ️](https://europeanremote.com/teams/stackoverflow) |
 |Stripe 🇺🇸 |Online payments|🖥 Ruby, Scala, Go, GraphQL 🎨 React, Flow ☁️ AWS| ❌ | ❌ |[ℹ️](https://europeanremote.com/teams/stripe) |
 |SUSE 🇩🇪 |Operating Systems|🖥 Python, Django, C, Golang 🎨 JavaScript, jQuery, ☁️ AWS, Azure, GCP| ❌ | ❌ |[ℹ️](https://europeanremote.com/teams/suse) |
+|Synthesia 🇬🇧 |Generative AI video|🖥 Python (Flask), Node.js, C++ 🎨 React, Typescript ☁️ AWS serverless| ❌ | ✅ |[ℹ️]([https://europeanremote.com/teams/suse](https://www.synthesia.io/careers)) |
```

Comment:
Thanks for your contribution.

The link is not quite right but I'll fix it in the next iteration.

Reply:
OMG you are right 🙈 I hope I did not cause too much of an inconvenience.
Thanks for merging.